src/platform/tigerlake/include/platform/lib/memory.h

```diff
@@ -261,7 +261,7 @@
 	HEAP_RT_COUNT1024 * 1024 + HEAP_RT_COUNT2048 * 2048 + \
 	HEAP_RT_COUNT4096 * 4096)
 
-#define HEAP_BUFFER_SIZE	0x50000
+#define HEAP_BUFFER_SIZE	0x80000
```

Comment:
why is this needed? to compensate for the resume latency?
That looks like a very large step in terms of allocation.

Reply:
The commit messages says " Thanks to this change we can provide KPB more space for history data"